ops/services/okta-app/_var.tf

```diff
@@ -28,6 +28,40 @@ variable "redirect_urls" {
   ]
 }
 
+variable "trusted_origin_preview_urls" {
+  description = "Trusted origin listing for the Okta Preview environment. Data format is [Name, URL, [Scopes]]"
+  default = [
+    ["localhost", "https://localhost.simplereport.gov1", ["CORS, REDIRECT"]],
```

Comment:
Potential typo? I don't think there should be a 1 at the end?

Reply:
Great catch! That would be a copypasta error on my part. That should be fixed, now!